app/src/main/java/intellistart/interviewplanning/controllers/nats/slot/event/NatsEventSubscriber.kt

```diff
@@ -0,0 +1,20 @@
+package intellistart.interviewplanning.controllers.nats.slot.event
+
+import intellistart.interviewplanning.NatsSubject
+import intellistart.interviewplanning.output.pubsub.slot.SlotUpdatedEvent
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Sinks
+
+@Component
+class NatsEventSubscriber(private val connection: Connection) {
+
+    fun subscribe(slotId: String): Flux<SlotUpdatedEvent> {
+        val sink = Sinks.many().multicast().onBackpressureBuffer<SlotUpdatedEvent>()
+        connection.createDispatcher { message ->
+            sink.tryEmitNext(SlotUpdatedEvent.parseFrom(message.data))
```

Comment:
`parseFrom` could throw. So, we should catch possible exception and emit the exception to sink:
```kotlin
            runCatching {
                SlotUpdatedEvent.parseFrom(message.data)
            }
                .onSuccess(sink::tryEmitNext)
                .onFailure(sink::tryEmitError)
```